docs/configuration.md

```diff
@@ -2670,6 +2670,9 @@ Spark uses [log4j](http://logging.apache.org/log4j/) for logging. You can config
 `log4j.properties` file in the `conf` directory. One way to start is to copy the existing
 `log4j.properties.template` located there.
 
+By default, spark adds to the MDC 2 records: `appId` and `appName` you can add that to your patternLayout `%X{appId}` in order to print in the logs
```

Comment:
do you intend to point it as a class or a word: `patternLayout`?

Reply:
log4j configuration.